crates/harp/src/pretty.rs

```diff
@@ -0,0 +1,191 @@
+//
+// pretty.rs
```

Comment:
oh or the original `harp::lgl_to_string()`

Reply:
The idea was to imply there is some kind of opinionated nature about how it converts to string, and that it is mainly useful for showing in the IDE in some place, rather than as some "definitive" way to convert this object to a string. But I can go back to `lgl_to_string()` and we can tweak in the future as needed